static/js/components/StaffLearnerInfoCard.js

```diff
@@ -22,21 +23,38 @@ const formatCourseRun = R.curry((title, run) => ({
   hasContactEmail: false,
 }));
 
-const formatCourseRuns = course => (
-  course
+const formatCourseInfo = course => (
+  [course
     .runs
     .filter(R.propSatisfies(s => s !== STATUS_OFFERED, 'status'))
-    .map(formatCourseRun(course.title))
+    .map(formatCourseRun(course.title)),
+    course,
+  ]
 );
 
+const pruneUnenrolledCourses = R.filter(([runs]) => (
+  !R.isEmpty(runs)
```

Comment:
I'm not sure - the behavior was there before, but I needed to implement it in a slightly different way because I now want the `renderCourseInfo` function to take `[formattedCourseRuns, course]` tuples instead of just a list of `formattedCourseRuns`. Before it was sufficient simply to use `R.flatten` to remove empty lists, but now instead of something like

```js
[
  [run1, run2],
  [],
]
```

I'm going to have something like

```js
[
  [[run1, run2], course1],
  [[], course2],
]
```

so I need to filter the list to exclude the elements where the first element of the `[runs, course]` tuple is empty. does that make sense?